src/components/HomePage.jsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable linebreak-style */
+import Navbar from './Navbar';
+import AddBook from './AddBook';
+import Categories from './Categories';
+
+const HomePage = () => (
+  <>
+    <Navbar />
+    <h1>Book Store</h1>
+    <Categories />
+    <AddBook />
```

Comment:
- [Optional] Great JOb for trying to arrange components, I think it would be better
if you create `books` component that holds `addbook` and `booklist` components and use it here instead of `addbook` component for better arrangement 👍